src/main/java/org/gridsuite/study/server/service/LoadFlowService.java

```diff
@@ -282,4 +230,123 @@ public List<LimitViolationInfos> getLimitViolations(UUID nodeUuid) {
         }
         return result;
     }
+
+    public LoadFlowParametersInfos getLoadFlowParameters(UUID parametersUuid) {
+        LoadFlowParametersInfos parameters;
+
+        String path = UriComponentsBuilder.fromPath(DELIMITER + LOADFLOW_API_VERSION + PARAMETERS_URI)
+            .buildAndExpand(parametersUuid).toUriString();
+        try {
+            parameters = restTemplate.getForObject(loadFlowServerBaseUri + path, LoadFlowParametersInfos.class);
+        } catch (HttpStatusCodeException e) {
+            if (HttpStatus.NOT_FOUND.equals(e.getStatusCode())) {
+                throw new StudyException(LOADFLOW_PARAMETERS_NOT_FOUND);
+            }
+            throw handleHttpError(e, GET_LOADFLOW_PARAMETERS_FAILED);
+        }
+        return parameters;
```

Comment:
You can return from the try block to be more simple
```suggestion
        final String path = UriComponentsBuilder.fromPath(DELIMITER + LOADFLOW_API_VERSION + PARAMETERS_URI)
            .buildAndExpand(parametersUuid).toUriString();
        try {
            return restTemplate.getForObject(loadFlowServerBaseUri + path, LoadFlowParametersInfos.class);
        } catch (final HttpStatusCodeException ex) {
            if (HttpStatus.NOT_FOUND.equals(ex.getStatusCode())) {
                throw new StudyException(LOADFLOW_PARAMETERS_NOT_FOUND);
            }
            throw handleHttpError(ex, GET_LOADFLOW_PARAMETERS_FAILED);
        }
```